File: 69A - Young Physicist.cpp
#include <iostream>
 
using namespace std;
 
int main()
{
    int n=0,x=0,y=0,z=0,xx=0,yy=0,zz=0;
    cin>>n;
    for(int i = 0 ; i < n ; i++)
    {
        cin>>x;
        xx=xx+x;
        cin>>y;
        yy=yy+y;
        cin>>z;
        zz=zz+z;
    }
    if(xx==0 && yy==0 && zz==0){
        cout<<"YES";
    }
    else{
       cout<<"NO"; 
    }
 
    return 0;
}

File: 69A - Young Physicist.java
import java.util.Scanner;
public class JavaApplication18 {
 
    public static void main(String[] args) {
        Scanner sc = new Scanner (System.in);
    int n = sc.nextInt();
    int x= 0 , y = 0 , z = 0 ; 
    while(n-->=1){
      x = x + sc.nextInt();
      y = y + sc.nextInt();
      z = z + sc.nextInt();
    }  
    if (x==0 && y ==0 && z == 0){
       System.out.println("YES");
    }
    else {
      System.out.println("NO");
    }
    }
    
}